sklearn/metrics/__init__.py

```diff
@@ -171,4 +173,6 @@
     "v_measure_score",
     "zero_one_loss",
     "brier_score_loss",
+    "precision_at_recall_k",
```

Comment:
I think that it would be better to call it `max_precision_at_recall_k` and `max_recall_at_precision_k` to make it obvious that this is the maximum that is taken.

Reply:
When I hear "precision_at_recall_k" I think of a single number singled out from the precision recall curve. (given a line, if I constrain X=x_i, then Y=y_i). If I agree with that logic then I think the original name is better to use.